tests/server_stubs/notebook_output_download_stub.py

```diff
@@ -0,0 +1,69 @@
+from flask import Flask, jsonify, request
+from flask.typing import ResponseReturnValue
+
+from kagglehub.http_resolver import NOTEBOOK_CURRENT_VERSION_FIELD
+from tests.utils import (
+    AUTO_COMPRESSED_FILE_NAME,
+    add_mock_gcs_route,
+    get_gcs_redirect_response,
+)
+
+app = Flask(__name__)
+add_mock_gcs_route(app)
+
+TARGZ_ARCHIVE_HANDLE = "testuser/zip-notebook/versions/1"
```

Comment:
nit: Surprised to see this since we expect zips and not tars?  Also not seeing `zip-notebook` used anywhere else in this PR?